### ISSUE TITLE
CORS failure

### ISSUE THREAD

**Issue by RReverser** [amtunlimited/average-response#4]
When going to https://aarontag.dev/average-response/ in the latest Firefox Developer Edition, I get the following in the console:

![image](https://user-images.githubusercontent.com/557590/96351349-9da52100-10b2-11eb-849d-26a41cd3bdb2.png)

**Comment by amtunlimited** [comment]
Thanks for reporting. Surprising that I haven't seen this yet

Are you still getting a number or is the ellipses not changing? I would
assume the result never loads

**Comment by RReverser** [comment]
Yeah just seeing ellipsis forever, which is why I opened the console.

**Comment by RReverser** [comment]
Oh... I see. Of course, it's uBlock Origin not liking the telemetry URL.

Probably doesn't count like a bug, but perhaps the page could at least catch the `fetch` failure and show an error + suggestion to disable the Adblocker instead of hanging forever?

![image](https://user-images.githubusercontent.com/557590/96353043-6e95ac00-10c0-11eb-9661-605b7b33907f.png)

**Comment by amtunlimited** [comment]
Huh. TIL that uBlock Origin blocking something looks like a CORS error.

Thanks for the heads up. I'll probably look at it tonight or tomorrow.
